stories/chats/src/BubbleMedia/hooks.tsx

```diff
@@ -162,9 +164,19 @@ export const useImageSize = (data: string | null) => {
     }, [maxImageSize.width, maxImageSize.height]);
 
     /**
-     * `data` has changed
+     * `originalSize` or `data` has changed
      */
     React.useEffect(() => {
+        if (originalSize && originalSizeRef.current) {
+            const newImageSize = getImageSize(
+                originalSizeRef.current?.width,
+                originalSizeRef.current?.height,
+                maxImageSize.width,
+                maxImageSize.height,
+            );
+            setImageSize(newImageSize);
+            return;
+        }
```

Comment:
Looks a little like a double-code from above. Possibly we are to move `originalSize` as a dependency there.... BUT I don't see where the `originalSize` value is really used here..

Reply:
`originalSize` value is used as the initial for originalSizeRef, so when we already know the image's dimensions we won't need to use Image.getSize() to get them